doc/source/dev/governance/governance.rst

```diff
@@ -262,7 +262,7 @@ conflict of interests include, but are not limited to:
 All members of the Council shall disclose to the rest of the Council any
 conflict of interest they may have. Members with a conflict of interest
 in a particular issue may participate in Council discussions on that
-issue, but must recuse themselves from voting on the issue.
+issue, but must recurse themselves from voting on the issue.
```

Comment:
not a big fan of a adding automated spell-checking if we'll have to ignore it making mistakes like this...

Reply:
rright -- I even ran into this particular one before -- `recuse` is indeed rare and I failed to spot it in review, sorry about that. I will whitelist it.